### ISSUE TITLE
[QUESTION] Unexpected encoded path?

### ISSUE THREAD

**Issue by samgarasx** [skrapeit/skrape.it#141]
I'm trying to use scrapeit for the first time and it's throwing an error like:

_Exception in thread "main" java.lang.IllegalArgumentException: unexpected encodedPath:_

```
val extracted = skrape(HttpFetcher) {
    request {
        url = config.getString("url")
    }
    extract { 
        htmlDocument {
            h1 { findFirst { text } }
        } 
    }
}
```

Also, I don't see the option to _ignoreContentType_, since maybe it's the problem.

**Comment by samgarasx** [comment]
Fixed, error is caused by not adding a `'/'` to the end of the Url

**Comment by christian-draeger** [comment]
Hey glad to hear you are giving skrape{it} a try.
This is a valid workaround for the HttpFetcher (problem should not exist for BrowserFetcher or AsyncFetcher).

It is already fixed on the master and will be part of the 1.1.0 release that will be published within the next days.
Since version 1.1.0 the "/" workaround should not be needed anymore.
